test/index.ts

```diff
@@ -17,12 +17,16 @@ const getExportName = (name: string): string => {
   return name.substring(0, 1).toLowerCase() + name.substring(1)
 }
 
+const shouldExport = (name: string): boolean => {
+  return !(name in { index: null, LocalStorage: null })
```

Comment:
isn't it too specific?

Reply:
This test is just a dirty way to remember the export when adding a new module